Libraries/Contents/PublicationStatus.cs

```diff
@@ -8,9 +8,9 @@ namespace Lombiq.HelpfulLibraries.Libraries.Contents
     public enum PublicationStatus
     {
         /// <summary>
-        /// Only used for querying without regards to publication status or to indicate that the status was unset.
+        /// Only used for querying to return all regardless of publication status or to indicate that the status was unset.
         /// </summary>
-        Any,
+        All,

```

Comment:
Revideálnom kell, amit korábban mondtam, elnézést, ez tényleg jobb `Any`-nek. `All` lenne, ha az összes versiont visszaadná. A `PublicationStatusEnum.All` amúgy ezt is csinálja, legalábbis a dokumentációja szerint; ha tényleg, akkor viszont a GraphQL-lel kapcsolatban csak az `All`-t tudjuk használni, `Any`-t nem.

Reply:
![image](https://user-images.githubusercontent.com/4007293/104651719-53830a00-56b8-11eb-895a-947c49320d82.png)

Reply:
:D Nézd meg az OC kódban a `PublicationStatusEnum`-ot.